snap/hooks/configure

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
+# Detect failure mode with early core18
+if ! which snapctl > /dev/null; then
+    if [ -e "$SNAP_DATA/config_options" ]; then
+        echo "ERROR: 'snapctl' is missing, configuration will fail."
+        echo "    This should be fixed (in core18) soon."
+        echo "    Workaround: set app to one of:"
+        ls -1 $SNAP/apps/ | sed "s/^/        /"
+        echo "    by editing /var/snap/mir-kiosk-apps/current/config_options"
+        exit 1
+    fi
+fi
+
```

Comment:
I think that's unnecessary now.

Reply:
Yes

Reply:
Actually I was wrong, it's not in the stable `core18` base yet.